appium/lib/screen-object/accessors/element.rb

```diff
@@ -194,6 +193,10 @@ def get_element_by_text(value)
         end
       end
 
+      def get_element_children_by_class(class_name)
+        element.find_elements(:class, class_name)
```

Comment:
Why only class, why not pass the locator type and value as a hash and use to locate children elements?